### PR TITLE
fix: add navbar to mobile view

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Added the navbar to mobile view on and added the nav drawer

- [Add nav to mobile view](https://3.basecamp.com/3105655/buckets/26787371/todos/4752596611) 
- [Add the nav drawer](https://3.basecamp.com/3105655/buckets/26787371/todos/4752596471)

# How should this PR be QA Tested?

- [x] On mobile view a navbar should be seen on top of the screen. 
- [x] On mobile view clicking on the hamburger menu opens the NavigationDrawer
- [x] On desktop view clicking on the right button within the NavigationDrawer should expand it.

# Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { Meta, Story } from '@storybook/react'
+import { noop } from 'lodash'
+import { MockedProvider } from '@apollo/client/testing'
+import { AuthUser } from 'next-firebase-auth'
+import { simpleComponentConfig } from '../../../libs/storybook'
+import { GET_ME, NavigationDrawerProps } from './NavigationDrawer'
+import { NavigationDrawer } from '.'
+
+const NavigationDrawerStory = {
+  ...simpleComponentConfig,
```
Comment (storyworks) [inline apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.stories.tsx:11]: Navigation usually gets more complex, might be safer to use journeysAdmin config..

File: apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx [hunk 2]
```diff
@@ -34,18 +23,6 @@ export interface PageWrapperProps {
...
-      lastName
-      email
-      imageUrl
-    }
-  }
-`
-
 export function PageWrapper({
   backHref,
   showDrawer,
```
Comment (storyworks) [inline apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx:28]: is showDrawer redundant now? When should we ever hide it?

**Comment by storyworks** [comment]
For some reason, PageWrapper "Complete" scenario doesn't show the black app bar...

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                }}
+              />
+            </ListItemIcon>
+            {open && <Typography variant="h5">Discover</Typography>}
+          </ListItemButton>
+        </Link>
+      </List>
+      {AuthUser != null && data?.me != null && (
+        <>
+          <Divider variant="middle" sx={{ borderColor: '#444451' }} />
```
[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx:155]: The default divider color doesn't match up with the Figma designs. I found that the color used is secondary.main. But removed any colors for now until this has been decided on

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/UserMenu/UserMenu.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  const handleProfileClose = jest.fn()
+  const signOut = jest.fn()
+  it('should render the menu', () => {
+    const { getByText, getByRole } = render(
+      <MockedProvider>
+        <UserMenu
+          user={{
+            __typename: 'User',
+            id: 'userId',
+            firstName: 'Amin',
```
[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/UserMenu/UserMenu.spec.tsx:16]: I found that this was being used in other places e.g. AccessDialog so decided to just stick with it XD

File: apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx (hunk 2 above)
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx:28]: We don't show the drawer on the JourneyList

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (storyworks) [reply apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx]: Oh! yeah I see - if it matches `secondary.main` use that then. When the color for `divider` was picked, seems like they only consider it for light backgrounds - not dark..

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ import taskbarIcon from '../../../../public/taskbar-icon.svg'
 import { GetMe } from '../../../../__generated__/GetMe'
 import { UserMenu } from './UserMenu'
 
-const drawerWidth = '237px'
+const DRAWER_WIDTH = '237px'
```
Comment (storyworks) [inline apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx:25]: Great thinking marking this a static variable - same could be done for that '72px' Can't remember where I saw it.

Comment (storyworks) [reply apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx:25]: And nice work fixing that animation - so the icons stay in the same place

Comment (storyworks) [reply apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.tsx:25]: My only other thing would be to double check with Tatai if he still wants to use `useMediaQuery` at some point before he went around removing all instances of it

**Review by storyworks — APPROVED** [review]

**Review by storyworks — APPROVED** [review]

File: apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx [hunk 6]
```diff
@@ -50,7 +50,7 @@ export function PageWrapper({
           }
         }}
       >
-        {!smUp ? (
+        {!smUp && (title === 'Journeys' || title === 'Journey Details') ? (
```
Comment (storyworks) [inline apps/journeys-admin/src/components/PageWrapper/PageWrapper.tsx:53]: Is this tested in VR?

**Pull request closed without merging**